pulsar-testclient/src/main/java/org/apache/pulsar/testclient/ManagedLedgerWriter.java

```diff
@@ -164,19 +175,23 @@ public static void main(String[] args) throws Exception {
         // Now processing command line arguments
         String managedLedgerPrefix = "test-" + DigestUtils.sha1Hex(UUID.randomUUID().toString()).substring(0, 5);
 
+        if (arguments.metadataStoreUrl == null) {
+            arguments.metadataStoreUrl = arguments.zookeeperServers;
+        }
+
         ClientConfiguration bkConf = new ClientConfiguration();
         bkConf.setUseV2WireProtocol(true);
         bkConf.setAddEntryTimeout(30);
         bkConf.setReadEntryTimeout(30);
         bkConf.setThrottleValue(0);
         bkConf.setNumChannelsPerBookie(arguments.maxConnections);
-        bkConf.setZkServers(arguments.zookeeperServers);
+        bkConf.setMetadataServiceUri(arguments.metadataStoreUrl);
```

Comment:
It seems that only the param `arguments.metadataStoreUrl` is valid, right?

Reply:
Yes.